README.md

```diff
@@ -144,6 +144,24 @@ au TextChanged * ++nested call Delay_My_Write()
 au TextChangedI * ++nested call Delay_My_Write()
 ```
 
+### Configuring Firenvim to be triggered manually only
+
+To have Firenvim work this way, you need to deactivate the auto-triggering of Firenvim in all cases.
+
+```vimscript
+" in your init.vim
+let g:firenvim_config = {
+  \ 'localSettings': {
+    \ '.*': {
+      \ 'selector': '',
+      \ 'priority': 0,
+    \ },
+  \ }
+\ }
```

Comment:
This is really similar to the "disable Firenvim everywhere but enable it on GitHub" example config that is shown lines 106-119. Would this have really helped you find the information you're looking for? (I don't mean this in a disparaging way - I'm just concerned about duplicating information in the documentation because the bigger it is, the less time people will spend reading it).

Perhaps adding a line like "Note that the <C-e> shortcut will still work on all websites" would be enough? Wat do you think?

Reply:
It makes sense to not duplicate, but I think I still wouldn't have understood by only reading the "disable everywhere except Github" example :thinking: 

Maybe we can rephrase it and keep it here indeed. I'll think this through!